cmd/juju/application/deployer/deployer.go

```diff
@@ -98,6 +99,19 @@ func (d *factory) setConfig(cfg DeployerConfig) {
 	d.flagSet = cfg.FlagSet
 }
 
+func isLocalSchema(u string) bool {
```

Comment:
I prefer private utility methods like these to be lower down the file, at least below their usage so that when browsing, the important/exported concepts are first.

Reply:
Yes, I'll fix that!